### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by dishantagnihotri**
### Description
Please include a concise summary of the changes you've made and reference the related issue (if any). Provide context for your code changes.

### Type of Change
Please select the appropriate option by placing an 'x' inside the brackets:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Design the page (New page design given by client )
- [ ] making API's

### How Has This Been Tested?
Please describe the tests you've conducted to verify your code changes. Provide instructions so others can reproduce the tests. List any relevant details for your test configuration. If possible attach a loom video
- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,12 +36,11 @@ <h1 class="font_0 wixui-rich-text__text" style="line-height:normal; text-align:c
...
   window.document.addEventListener('vinoshipper:loaded', () => {
     window.Vinoshipper.init(4855, {
       theme: 'gray',
     });
   });
 </script>
 
-<script src="https://vinoshipper.com/injector/index.js" type="text/javascript"></script>
\ No newline at end of file
+<script src="https://vinoshipper.com/injector/index.js" type="text/javascript"></script>
```
Comment (koushalrajput) [inline index.html:46]: Please < script/> tag close

**Review by dshant — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -36,12 +36,11 @@ <h1 class="font_0 wixui-rich-text__text" style="line-height:normal; text-align:c
 <div class="vs-products" data-vs-cards="true"></div>
```
Comment (koushalrajput) [inline index.html:36]: <div/> tag close.

**Review by koushalrajput — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (dishantagnihotri) [inline index.html:36]: Pure HTML h :) 
Not possible

File: index.html (hunk 1 above)
Comment (dishantagnihotri) [inline index.html:46]: Pure HTML h :)
Not possible

**Review by nishtha0001 — APPROVED** [review]

**Review by koushalrajput — APPROVED** [review]

**Review by koushalrajput7 — APPROVED** [review]

**Pull request merged**